modules/channel-slack/src/backend/client.ts

```diff
@@ -227,6 +224,62 @@ export class SlackClient {
     next(undefined, false)
   }
 
+  private async applyChannelEffects(event: sdk.IO.OutgoingEvent) {
```

Comment:
Effects is a general name to what can be applied on a payload to change the behavior. It's not really a mapper because there's already a "convertPayload" method.

It's not really a renderer because it may send a payload (eg: send typing), but it may also change the payload (add a markdown flag). Couldn't find a better name